lib/from_openstudio/model.rb

```diff
@@ -131,6 +133,10 @@ def self.energy_properties_from_model(openstudio_model)
       hash[:materials] = materials_from_model(openstudio_model)
       hash[:construction_sets] = []
       hash[:construction_sets] = constructionsets_from_model(openstudio_model)
+      hash[:schedule_type_limits] = []
+      hash[:schedule_type_limits] = schedtypelimits_from_model(openstudio_model)
+      hash[:schedules] = []
+      hash[:schedules] = scheduleruleset_from_model(openstudio_model)
```

Comment:
The "code smell" doesn't quite seem right here you shouldn't have to set the variable to an empty list if you are immediately going to change it. This should probably be:
```
hash[:schedule_type_limits] = schedtypelimits_from_model(openstudio_model)
hash[:schedules] = scheduleruleset_from_model(openstudio_model)
```